src/main/java/com/codingnomads/betty/configurations/TwitterConfigurer.java

```diff
@@ -0,0 +1,39 @@
+package com.codingnomads.betty.configurations;
+
+import org.springframework.stereotype.Component;
+import twitter4j.Twitter;
+import twitter4j.TwitterFactory;
+import twitter4j.conf.ConfigurationBuilder;
+
+@Component
+public class TwitterConfigurer {
+
+    private Twitter twitter;
+    private TwitterFactory twitterFactory;
+
+    public TwitterConfigurer() {
+
+        buildTwitterConfiguration();
+    }
+
+    public Twitter getTwitter() {
+        return twitter;
+    }
+
+    private void buildTwitterConfiguration() {
+
+        ConfigurationBuilder configurationBuilder = new ConfigurationBuilder();
+
+        configurationBuilder
+                .setDebugEnabled(true)
+                .setOAuthConsumerKey("hv8cRVlIIsTK44fly4Bdd6sRn")
```

Comment:
We need to find a way to not expose the keys on the code. Maybe use Travis or find a different way?